ext/REPLExt/completions.jl

```diff
@@ -37,7 +37,13 @@ function complete_expanded_local_dir(s, i1, i2, expanded_user, oldi2)
     cmp = REPL.REPLCompletions.complete_path(s, i2, shell_escape=true)
     cmp2 = cmp[2]
     completions = [REPL.REPLCompletions.completion_text(p) for p in cmp[1]]
-    completions = filter!(x -> isdir(s[1:prevind(s, first(cmp2)-i1+1)]*x), completions)
+    completions = filter!(completions) do x
+        try
+            isdir(s[1:prevind(s, first(cmp2)-i1+1)]*x)
```

Comment:
Shouldn't this function return `true` or `false`? Does it always do that now or is the fall through returning `nothing` which would error in `filter!`?

Reply:
It throws if it doesn't have perms to stat the path

Reply:
Yes, and it should return `false` in that case, but right now it isn't I think?

Reply:
@vtjnash perhaps you have an opinion on what to do here?